package.json

```diff
@@ -93,12 +93,11 @@
     "@react-native-firebase/storage": "11.5.0",
     "@react-native-masked-view/masked-view": "^0.2.6",
     "@react-native-picker/picker": "^2.4.2",
-    "@react-navigation/compat": "^5.3.20",
-    "@react-navigation/core": "^6.2.1",
     "@react-navigation/drawer": "^5.12.9",
+    "@react-navigation/elements": "^1.3.6",
     "@react-navigation/material-top-tabs": "^5.3.19",
-    "@react-navigation/native": "^5.9.8",
-    "@react-navigation/stack": "^5.14.9",
+    "@react-navigation/native": "^6.0.13",
```

Comment:
do we need to upgrade this now? reading the docs you linked, https://reactnavigation.org/docs/upgrading-from-5.x/#note-on-mixing-react-navigation-5-and-react-navigation-6-packages it seems like there are some breaking changes when upgrading this particular package. for example, https://reactnavigation.org/docs/upgrading-from-5.x/#no-more-state-property-on-the-route-prop - i think we are using this in the NavigationWrapper. if we're already going to mix v5 and v6 navigators, can we minimise the amount of unnecessary upgrades so that we can see more easily which code changes are related to which upgrades? it'd be nice to avoid sneaking in unrelated upgrades in the same PR 😅 we can always create a PR especially for an upgrade

Reply:
Regarding the documentation with `route.state` not being present, It is still present in v6 even if the docs say it isn't. We do use it in our navigation wrapper to get the getActiveRouteName recursively and this appears to be core to how we're managing our navigation. Left another comment about why we should also upgrade `@react-navigation/native` [here](https://github.com/valora-inc/wallet/pull/3004#discussion_r1014341590).